### PR TITLE
use tabset for Maven/sbt in dev-env.adoc and template.adoc, #343

### PR DESCRIPTION
**Pull request by patriknw**
* because there hasn't been any source code choice yet it could be easy to miss
  the language choice
* leaving later sbt/mvn snippets as is, but when/if we add gradle we will probably
  use tabs in all places

References #343

If trying this you should use `make local-preview` instead of opening the files in browser.

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc [hunk 1]
```diff
@@ -1,11 +1,15 @@
 = I: Set up your development environment
 :page-supergroup-java-scala: Language
+:page-supergroup-maven-sbt: Build tool
```
Comment (octonato) [inline docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc:3]: Two selectors will make it confusing, IMO. 

Choosing `scala` should imply `sbt` and choosing `java` implies `mvn`. 

That's the combination we have in the templates. I think the page should stick with it.

Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc:3]: Then the tab titles can’t be Maven/sbt but ok I’ll change

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc]: You mean they need to be Java/Scala? 

Then we will have a problem when we introduce gradle. 

Maybe we should have no tabs for the build tool and change the language selector to a better position. See https://github.com/lightbend/antora-ui-lightbend-theme/issues/73

Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc]: That's what I said in the ticket, from the beginning. We can use Java/Scala for now, and I think the tabs serve the purpose on these initial pages to make it more evident that there is a choice.

**Review by octonato — APPROVED** [review]

**Pull request merged**